### PR TITLE
Colorslider

### PR DESCRIPTION
**Pull request by mischnic**
Closes https://github.com/adobe/react-spectrum/issues/1119

Includes https://github.com/adobe/spectrum-css/pull/1034

## TODO

- [x] Hook up labels. vertical labels?
- [x] Tests
- [x] decide on API

### CODE REVIEW

File: packages/@react-spectrum/color/src/ColorSlider.tsx [hunk 1]
```diff
@@ -0,0 +1,136 @@
...
+    case 'green':
+    case 'blue':
+      return value.withChannelValue('alpha', 1);
+    case 'alpha': {
+      return value;
+    }
+    default:
+      throw new Error('Unknown color channel: ' + channel);
+  }
+}
```
Comment (mischnic) [inline packages/@react-spectrum/color/src/ColorSlider.tsx:45]: Move this to stately?

File: packages/@react-aria/color/src/useColorSlider.tsx [hunk 2]
```diff
@@ -0,0 +1,33 @@
...
+    index: 0
+  }, state);
+
+  return {
+    containerProps,
+    trackProps,
+    inputProps,
+    thumbProps
+  };
+}
```
Comment (mischnic) [inline packages/@react-aria/color/src/useColorSlider.tsx:33]: There's not much going on here...

**Comment by mischnic** [comment]
I couldn't find a design for this....

Bildschirmfoto 2020-10-05 um 18 42 40

File: packages/@react-spectrum/color/src/ColorSlider.tsx (hunk 1 above)
Comment (devongovett) [reply packages/@react-spectrum/color/src/ColorSlider.tsx:45]: I think that makes sense.

File: packages/@react-spectrum/color/src/ColorSlider.tsx [hunk 3]
```diff
@@ -0,0 +1,136 @@
...
+
+function getBackground(state: ColorSliderState, channel: ColorChannel, vertical: boolean, direction: Direction) {
```
Comment (devongovett) [inline packages/@react-spectrum/color/src/ColorSlider.tsx:47]: I think this function makes sense in aria. Doesn't seem to spectrum-specific to me...

**Review by devongovett — APPROVED** [review]

**Pull request merged**